src/main/utils/jfrogAppsConfig/jfrogAppsConfig.ts

```diff
@@ -0,0 +1,161 @@
+import * as fs from 'fs';
+import yaml from 'js-yaml';
+import * as path from 'path';
+import { ScanType } from '../../scanLogic/scanRunners/analyzerModels';
+import { AnalyzerUtils } from '../../treeDataProviders/utils/analyzerUtils';
+import { ExcludeScannerName, JFrogAppsConfigSchema, Module, SastScanner, Scanner } from '../../types/jfrogAppsConfigSchema';
+import { Configuration } from '../configuration';
+
+export class JFrogAppsConfig {
+    public static JFROG_APP_CONFIG_VERSION: string = '1.0';
+
+    private _version: string = JFrogAppsConfig.JFROG_APP_CONFIG_VERSION;
+    private _modules: AppsConfigModule[] = [];
+
+    constructor(workspace: string) {
+        let appConfigPath: string = path.join(workspace, '.jfrog', 'jfrog-apps-config.yml');
+        if (fs.existsSync(appConfigPath)) {
+            let jfrogAppsConfig: JFrogAppsConfigSchema = yaml.load(fs.readFileSync(appConfigPath, 'utf8')) as JFrogAppsConfigSchema;
+            this._version = jfrogAppsConfig.version;
+            if (jfrogAppsConfig.modules) {
+                for (let module of jfrogAppsConfig.modules) {
+                    this._modules.push(new AppsConfigModule(module));
+                }
+            }
+        }
+        // If no modules provides, push a default module
+        if (this._modules.length === 0) {
+            this._modules.push(new AppsConfigModule({ source_root: workspace } as Module));
```

Comment:
What about the default scanners?

Reply:
The scanners' configuration is the thing that makes them not-default